server/app/template_globals.py

```diff
@@ -28,3 +28,21 @@ def url_page(url: str, page: int, param_name: str):
 def datetime(value):
     """Given a datetime, it format the out to use yyyy-mm-dd hh:mm:ss format"""
     return value.isoformat(" ", "seconds")
+
+@app.template_filter()
+def rating_icon(value):
+    base = "product-rating glyphicon "
+    if value < 0.2:
+        return base + "glyphicon-pawn text-danger"
```

Comment:
I think using the chess icons is cryptic. For a non-players, I don't know whether knight or bishop is better (I would think knight, as its movement is more uncommon, so could perform attacks others couldn't). I like the coloring pattern, though. I think it would be much cleaner to present this as a color-coded number (17% on red background, 87% on green etc). Also, when showing the actual value, you can easily compare between observations on a list, without the need to hover your cursor.

Also, it's much more readable on devices without cursor (tablets, phones).

Reply:
On a second thought, if you like the icons concept, I think it would be ok to show both the icon and the percent value in the list the same way how it's presented in a single observation.

Sadly, there are no better icons in glyphicon. I was looking for something like wifi strength or battery charge, but there isn't any :(

Reply:
Oh, you're right. Not everyone needs to know the value of chess pieces and symbolic icons (queen is very similar to king).
I think that put icon and percentage alongside is ugly. I changed the solution to use only value in border.

Reply:
Looks great!